fetch.bs

```diff
@@ -4296,6 +4299,16 @@ steps:
    <a for="fetch params">timing info</a>.</p></li>
   </ol>
  </li>
+ <li>
+  <p>Otherwise, if <var>response</var>'s <a for=response>URL</a>'s <a for=url>scheme</a> is
+  "<code>https</code>", then set <var>response</var>'s <a for=response>timing info</a>'s
+  <a for="fetch timing info">server timing headers</a> to the result of
+  <a for="header list">getting, decoding, and splitting</a> `<code>Server-Timing</code>` from
+  <var>response</var>'s <a for=response>header list</a>.
+
+  <p>The user agent may decides to expose `<code>Server-Timing</code>` headers to
+  "<code>http</code>" requests as well.
```

Comment:
This should be a secure context check of sorts as these headers are fine on `localhost`.

Reply:
@annevk I could use some guidance here :)

Reply:
Can't we check if request's client is a secure context? (With that referencing the definition in HTML.) That should account for `localhost` as well as nested contexts and such.

Reply:
Gotcha. done